__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require("supertest");
+const app = require("../app/app");
+const db  = require("../db/connection");
+const dataTest = require("../db/data/test-data/index")
+const seed = require("../db/seeds/seed")
+
+beforeEach(() => {
+    return seed(dataTest);
+})
+
+afterAll(() => {
+    return db.end();
+})
+
+describe('GET /api/categories', () => {
+     test('/api/categories should return 200 status and array of categories ', () => {
+        return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((response) => {
+            expect(Array.isArray(response.body.Categories)).toBe(true);
+            expect(response.body.Categories[0]).toHaveProperty("slug");
```

Comment:
You could test each object in the array looks as it should using a forEach here - that would be a stronger test than only checking the first element in the array. 👍